core/src/martian/encoders.cljc

```diff
@@ -36,8 +37,15 @@
        (js->clj v :keywordize-keys key-fn))))
 
 #?(:clj
-   (defn multipart-encode [body]
-     (mapv (fn [[k v]] {:name (name k) :content v}) body)))
+   (defn multipart-encode
+     ([body]
+      (mapv (fn [[k v]]
+              {:name (name k) :content (multipart/coerce-content v)})
+            body))
+     ([body pass-pred]
+      (mapv (fn [[k v]]
+              {:name (name k) :content (multipart/coerce-content v pass-pred)})
+            body))))
```

Comment:
Here's an updated `multipart-encode` encoder fn for "multipart/form-data" requests, that now also does `:content` values coercion (so that they comply with the RFC specification). Allows the custom content types to "pass through" as is (the `pass-pred` parameter). Yet, it's backward compatible with the previous version.